packages/@ember/-internals/glimmer/lib/component.ts

```diff
@@ -39,9 +39,14 @@ export const BOUNDS = symbol('BOUNDS');
   The easiest way to create a `Component` is via
   a template. If you name a template
   `app/templates/components/my-foo.hbs`, you will be able to use
-  `{{my-foo}}` in other templates, which will make
+  `<MyFoo />` or `{{my-foo}}` in other templates, which will make
   an instance of the isolated component.
 
+  ```app/templates/components/my-foo.hbs
+  <PersonProfile @person={{this.currentUser}} />
+  ```
+  or
+
   ```app/templates/components/my-foo.hbs
   {{person-profile person=currentUser}}
```

Comment:
can you update the original samples for `this.currentUser` usage as well?

Reply:
Done